content/tutorials/demo-ably-tower-defense.textile

```diff
@@ -0,0 +1,40 @@
+---
+ably_product: reactor
```

Comment:
This doesn't use reactor, should this just not be set?

Reply:
I would also like to add: Reactor does not exist as a product anymore: it's been removed (as a word) from the Dashboard (it's now generally referred to as Integrations), and the docs. If we can, we should always try to avoid using the term "Reactor". Thanks.

Reply:
this is legacy un-used (ignore)